app/models/organisation.rb

```diff
@@ -186,6 +186,7 @@ def foi_contacts
 
   before_destroy { |r| r.destroyable? }
   after_save :ensure_analytics_identifier
+  after_commit :publish_services_and_information_content_item
```

Comment:
I've moved this to an async worker which is triggered in the respective controller